controls/radpdfviewer/features/find.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Find
+page_title: Find
+description: Find
+slug: radpdfviewer-find
+tags: find, findall, findprevious
+published: True
+position: 6
+---
+
+# Find
+
+__RadPdfViewer__ allows flexible searching in a loaded PDF document. The following article describes the available find methods, as well as how to use the respective for the feature API.
+
+## Using UI
```

Comment:
If you would like to only connect the articles, you can use notes instead of adding a new section for the link only.